src/content-panel.tsx

```diff
@@ -103,12 +104,18 @@ class Root extends React.Component {
   render() {
     const teamContent = (
       <div className="tt_main_container" onMouseEnter={this.updateCurrentScale}>
-        <h4 className="sub-title">Team types:</h4>
+        <h4 className="sub-title">Fundamental Team types:</h4>
```

Comment:
Could we arrange the shapes like this?

![image](https://user-images.githubusercontent.com/73941040/152135460-139bac93-20e5-41ed-8141-27472f4630c0.png)

Reply:
UI updated here: c72e216 ✅ 

**Preview**

<img width="455" alt="image" src="https://user-images.githubusercontent.com/77609814/154600385-3bb7ac92-a331-45c0-b1d7-0f5fd01b861f.png">